templates/openstackplaybookgenerator/bin/create-playbooks.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+set -eux
+
+unset OS_CLOUD
+export OS_AUTH_TYPE=none
+export OS_ENDPOINT="http://{{ .HeatServiceName }}:8004/v1/admin"
+
+HEAT_COUNT=0
+until openstack stack list &> /dev/null || [ "$HEAT_COUNT" -gt 180 ]; do
+  HEAT_COUNT=$(($HEAT_COUNT + 1))
+  echo "waiting for Heat to start..."
+  sleep 2
+done
+
+# if ROLES_FILE is set we overwrite the default t-h-t version
+ROLES_FILE="{{ .RolesFile }}"
+if [ -n "$ROLES_FILE" ]; then
+  cp /home/cloud-admin/config-custom/$ROLES_FILE $TEMPLATES_DIR/roles_data.yaml
+fi
+
+TEMPLATES_DIR=$HOME/tripleo-deploy-test/tripleo-heat-installer-templates
```

Comment:
Should `TEMPLATES_DIR` be defined before line 19?  It looks like it's used on 19 but isn't actually set until line 22.  Or perhaps it's passed to the container as an env var, but I just thought I'd check.

Reply:
This should be fixed in the latest patch.